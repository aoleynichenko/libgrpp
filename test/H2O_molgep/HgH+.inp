-- Hg2+ ion
-- HF calculation using gatchina ECP and basis set

title = "Hg2+ ion recp/hartree-fock test"

molecule = {
    {"Hg",  0, 0, 0, basis='hg-small.bas', ecp='hg-small.bas'},
    {"H",   0, 0, 4, basis='cc-pvdz.bas'}
}

wf.occ = 18
scf.print_orbs = 'no'
scf.use_grecp = false

run('scf')

