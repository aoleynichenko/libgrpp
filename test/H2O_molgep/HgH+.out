
		*************************************************************
		**                          F S C C                        **
		**         An Ab Initio Electronic Structure Package       **
		**                       Version 0.1.0                     **
		*************************************************************


 Authors (in alphabetic order):
   Ephraim Eliav           | Relativistic Fock-Space CC
   Andrei Zaitsevskii      | Finite-Field TDM
   Alexander Oleynichenko  | General design, driver program, FSCC, Finite-Field TDM
   Alexander Petrov        | Properties integrals transformation
   Anatoly Titov           | GRECP integrals, interface to MOLCAS-1986
   Uzi Kaldor              | Basic Fock-Space CC routines

 This release uses parts of the MOLCAS-1986 code:
   MOLECULE by Jan Almloef
   INTORD   by Bjorn Roos
   SUPCON   by Per-Aake Malmquist
   SCF      by Per-Olof Widmark
   MONSTR   by Bjorn Roos, Per-Olof Widmark and Anatoly Titov
   MOLPRO   by A. J. Sadlej
   FASTIO I/O subsystem by Per-Olof Widmark


 This is an experimental code. The authors accept no responsibility 
 for the performance of the code or for the correctness of the results.


   Compiled    | Oct 15 2021 19:42:17
   Compiler    | Intel C Compiler 2021
   Lua version | Lua 5.3
   Date        | Sun Oct 17 15:54:40 2021
   Host        | alexander
   OS          | Linux 5.10.42-1-MANJARO
   Hardware ID | x86_64


-- current .fsccrc settings:
binaries = {
	molgep = "molgep.x",
	supcon = "supcon.x",
	intord = "intord.x",
	scf = "scf.x",
	monstr = "monstr.x",
	mp2    = "mp2.x",
	ccsort_00 = "csdseltson.x",
	cc_00     = "csdson.x"
	ccsort_01 = "csvseltson.x",
	cc_01     = "csvson.x"
	ccsort_02 = "csuseltson.x",
	cc_02     = "csuson.x"
	ccsort_10 = "chvseltson.x",
	cc_10     = "chvson.x"
	ccsort_20 = "chuseltson.x",
	cc_20     = "chuson.x"
	ccsort_11 = "chpqseltson.x",
	cc_11     = "chpson.x"
}

Input file: HgH+.inp

========================== echo of input file =========================
-- Hg2+ ion
-- HF calculation using gatchina ECP and basis set

title = "Hg2+ ion recp/hartree-fock test"

molecule = {
    {"Hg",  0, 0, 0, basis='hg-small.bas', ecp='hg-small.bas'},
    {"H",   0, 0, 4, basis='cc-pvdz.bas'}
}

wf.occ = 18
scf.print_orbs = 'no'
scf.use_grecp = false

run('scf')

=======================================================================

opening ecp file : hg-small.bas

  rec.unit = 4 bytes
 *****************************************************************
                           M O L E C U L E
           WRITTEN BY JAN ALMLOF, UNIVERSITY OF LUND, SWEDEN.
                    INCLUDES UPDATES TO AUGUST 1986
 *****************************************************************

 ***  START OF CALCULATION *** TIME NOW      0.000
  Data from MolDat? "y" or "n": 
  Word for MolDat: 
 &MOLECU
  nml O'key 
 MAX TIME =    999.00000
 SAFETY INTERRUPT TIME =    999.00000
 DIFFERENT TYPES OF ATOMS:       3
 NUMBER OF SYMMETRY OPERATIONS:  0
 INTEGRALS LESS THAN  0.10E-07 WILL BE NEGLECTED

     ATOMIC TYPE NUMBER  1
     ---------------------

 BASIS SET TITLE AND COMMENT:
this basis set data are automatically generated by the FSCC driver      
                                                                        
 NUCLEAR CHARGE =  0.00
  1 SYMMETRY INDEPENDENT ATOMS
 HIGHEST ANGULAR MOMENTUM OF FUNCTIONS USED IS: G
  2 ORIGINAL AND  0  ADDED GROUPS OF CGTOS OF S TYPE
  2 ORIGINAL AND  0  ADDED GROUPS OF CGTOS OF P TYPE
  2 ORIGINAL AND  0  ADDED GROUPS OF CGTOS OF D TYPE
  2 ORIGINAL AND  0  ADDED GROUPS OF CGTOS OF F TYPE
  2 ORIGINAL AND  0  ADDED GROUPS OF CGTOS OF G TYPE
 NUCLEAR COORDINATES:
        Hg         0.000000000       0.000000000       0.000000000

 
 Nontyp=     1 Nucl.charge-N Fr.Sp.=     0.0000 N Fr.Sp.=     0.0000
 

     ATOMIC TYPE NUMBER  2
     ---------------------

 BASIS SET TITLE AND COMMENT:
this basis set data are automatically generated by the FSCC driver      
                                                                        
 NUCLEAR CHARGE =  1.00
  1 SYMMETRY INDEPENDENT ATOMS
 HIGHEST ANGULAR MOMENTUM OF FUNCTIONS USED IS: D
  1 ORIGINAL AND  0  ADDED GROUPS OF CGTOS OF S TYPE
  1 ORIGINAL AND  0  ADDED GROUPS OF CGTOS OF P TYPE
  1 ORIGINAL AND  0  ADDED GROUPS OF CGTOS OF D TYPE
 NUCLEAR COORDINATES:
        H          0.000000000       0.000000000       4.000000000

 
 Nontyp=     2 Nucl.charge-N Fr.Sp.=     1.0000 N Fr.Sp.=     0.0000
 

     ATOMIC TYPE NUMBER  3
     ---------------------

 BASIS SET TITLE AND COMMENT:
this ecp data are automatically generated by the FSCC driver            
                                                                        
 NUCLEAR CHARGE = 20.00
  1 SYMMETRY INDEPENDENT ATOMS
 HIGHEST ANGULAR MOMENTUM OF FUNCTIONS USED IS: X
 NUCLEAR COORDINATES:
        GR         0.000000000       0.000000000       0.000000000

 EFFECTIVE SO OPERATOR FORMAT:  GENERALIZED RECP INPUT
 U_{SO} = Delta U_{lj}(r)
 
 Nontyp=     3 Nucl.charge-N Fr.Sp.=    20.0000 N Fr.Sp.=     0.0000
 
Norb0=   80

          INTERNUCLEAR DISTANCES IN ANGSTROM UNITS
          ----------------------------------------




          Hg          0.00000   0.00000   0.00000

               H      1     DIST= 2.117
               GR     1     DIST= 0.000



          H           0.00000   0.00000   4.00000

               GR     1     DIST= 2.117



          GR          0.00000   0.00000   0.00000

             80 SYMMETRY ADAPTED BASIS FUNCTIONS 
 

          IRREDUCIBLE REPRESENTATION NUMBER 1

    1    1   Hg  S         1   1    1.00000
    2    2   Hg  S         1   2    1.00000
    3    3   Hg  X         1   3    1.00000
    4    4   Hg  Y         1   4    1.00000
    5    5   Hg  Z         1   5    1.00000
    6    6   Hg  X         1   6    1.00000
    7    7   Hg  Y         1   7    1.00000
    8    8   Hg  Z         1   8    1.00000
    9    9   Hg  XX        1   9    1.00000
   10   10   Hg  XY        1  10    1.00000
   11   11   Hg  XZ        1  11    1.00000
   12   12   Hg  YY        1  12    1.00000
   13   13   Hg  YZ        1  13    1.00000
   14   14   Hg  ZZ        1  14    1.00000
   15   15   Hg  XX        1  15    1.00000
   16   16   Hg  XY        1  16    1.00000
   17   17   Hg  XZ        1  17    1.00000
   18   18   Hg  YY        1  18    1.00000
   19   19   Hg  YZ        1  19    1.00000
   20   20   Hg  ZZ        1  20    1.00000
   21   21   Hg  XXX       1  21    1.00000
   22   22   Hg  XXY       1  22    1.00000
   23   23   Hg  XXZ       1  23    1.00000
   24   24   Hg  XYY       1  24    1.00000
   25   25   Hg  XYZ       1  25    1.00000
   26   26   Hg  XZZ       1  26    1.00000
   27   27   Hg  YYY       1  27    1.00000
   28   28   Hg  YYZ       1  28    1.00000
   29   29   Hg  YZZ       1  29    1.00000
   30   30   Hg  ZZZ       1  30    1.00000
   31   31   Hg  XXX       1  31    1.00000
   32   32   Hg  XXY       1  32    1.00000
   33   33   Hg  XXZ       1  33    1.00000
   34   34   Hg  XYY       1  34    1.00000
   35   35   Hg  XYZ       1  35    1.00000
   36   36   Hg  XZZ       1  36    1.00000
   37   37   Hg  YYY       1  37    1.00000
   38   38   Hg  YYZ       1  38    1.00000
   39   39   Hg  YZZ       1  39    1.00000
   40   40   Hg  ZZZ       1  40    1.00000
   41   41   Hg  XXXX      1  41    1.00000
   42   42   Hg  XXXY      1  42    1.00000
   43   43   Hg  XXXZ      1  43    1.00000
   44   44   Hg  XXYY      1  44    1.00000
   45   45   Hg  XXYZ      1  45    1.00000
   46   46   Hg  XXZZ      1  46    1.00000
   47   47   Hg  XYYY      1  47    1.00000
   48   48   Hg  XYYZ      1  48    1.00000
   49   49   Hg  XYZZ      1  49    1.00000
   50   50   Hg  XZZZ      1  50    1.00000
   51   51   Hg  YYYY      1  51    1.00000
   52   52   Hg  YYYZ      1  52    1.00000
   53   53   Hg  YYZZ      1  53    1.00000
   54   54   Hg  YZZZ      1  54    1.00000
   55   55   Hg  ZZZZ      1  55    1.00000
   56   56   Hg  XXXX      1  56    1.00000
   57   57   Hg  XXXY      1  57    1.00000
   58   58   Hg  XXXZ      1  58    1.00000
   59   59   Hg  XXYY      1  59    1.00000
   60   60   Hg  XXYZ      1  60    1.00000
   61   61   Hg  XXZZ      1  61    1.00000
   62   62   Hg  XYYY      1  62    1.00000
   63   63   Hg  XYYZ      1  63    1.00000
   64   64   Hg  XYZZ      1  64    1.00000
   65   65   Hg  XZZZ      1  65    1.00000
   66   66   Hg  YYYY      1  66    1.00000
   67   67   Hg  YYYZ      1  67    1.00000
   68   68   Hg  YYZZ      1  68    1.00000
   69   69   Hg  YZZZ      1  69    1.00000
   70   70   Hg  ZZZZ      1  70    1.00000
   71   71   H   S         1  71    1.00000
   72   72   H   X         1  72    1.00000
   73   73   H   Y         1  73    1.00000
   74   74   H   Z         1  74    1.00000
   75   75   H   XX        1  75    1.00000
   76   76   H   XY        1  76    1.00000
   77   77   H   XZ        1  77    1.00000
   78   78   H   YY        1  78    1.00000
   79   79   H   YZ        1  79    1.00000
   80   80   H   ZZ        1  80    1.00000
 
  NTRI=     3240

NUCLEAR REPULSION ENERGY = 0.5000000E+01

 ***  INPUT PROCESSING COMPLETE *** TIME NOW      0.000 *** TIME FROM LAST CALL      0.000
 ONEL & ONELH STARTED
 SUBROUTINE ONEL STARTED (OVERLAP INTEGRALS)
  ntri,idisk=         3240           8
 DAFILE: WRITING SOUT
 MATRIX ELEMENTS: S
 ONEL FINISHED
 SUBROUTINE ONELH STARTED (ONE-ELECTRON HAMILTONIAN INTEGRALS: POTENTIAL + KINETIC)
 DAFILE: WRITING HOUT AND TOUT
 MATRIX ELEMENTS: T
 MATRIX ELEMENTS: V
 ONELH FINISHED
 ONEL & ONELH FINISHED

 *****************************************************************
                            G R E C P
           Gaussian  Generalized RECP  integral  program
           Release 3.1 (September 1996) by Anatoli TITOV
           Petersburg Nuclear Physics Institute, RUSSIA
 *****************************************************************

 ONELEP STARTED
 MATRIX ELEMENTS: HCORE
 MATRIX ELEMENTS: AREP
 MATRIX ELEMENTS: SO-X
 MATRIX ELEMENTS: SO-Y
 MATRIX ELEMENTS: SO-Z
  WRITE ESOP-X INTS... IBUF2M=        3240 IDISK=         556
  WRITE ESOP-Y INTS... IBUF2M=        3240 IDISK=         584
  WRITE ESOP-Z INTS... IBUF2M=        3240 IDISK=         612
 ***  GRECP COMPLETED *** TIME NOW      1.000 *** TIME FROM LAST CALL      1.000
 One-electron part complete
 ***  ONE-ELECTRON INTEGRALS COMPLETE *** TIME NOW      1.000 *** TIME FROM LAST CALL      0.000
 Two-electron part is activated now 
  Total number of gaussian groups           13
 
 
      Processing group IA-IB-IC-ID              
            812846 INTEGRALS WRITTEN OUT IN    58 RECORDS

            812846 INTEGRALS WRITTEN OUT IN    58 RECORDS

 NUMBER OF EXTENTS NEEDED ON THE BASINT FILE IS     464
 ***  TWO-ELECTRON INTEGRALS COMPLETE *** TIME NOW      1.000 *** TIME FROM LAST CALL      0.000
 OKEJ 1

 *****************************************************************
 I N T O R D  AO integrals order program - MOLCAS 90-01-29
 *****************************************************************

  rec.unit = 4 bytes
  Data from MolDat? "y" or "n": 
  Word for MolDat: 

  NUMBER OF SYMMETRIES   1
  SYMMETRY SPECIES                       1
  NUMBER OF BASIS FUNCTIONS             80
  SYMMETRIES DELETED                     0
  PRINT PARAMETER      1
  SQUAR PARAMETER      1

  NUMBER OF MATRICES PER BLOCK IN EACH SYMMETRY
           8

  NUMBER OF BLOCKS IN EACH SYMMETRY
        405

  STARTING DISK ADDRESS FOR EACH SYMMETRY
          8

  TOTAL NUMBER OF BLOCKS ON DISK         405
  LENGTH OF BINS IN CORE               81920
  MAX LENGTH OF INTEGRAL BUFFER        25920
  CORE AREA USED FOR BINS           33501579
  NUMBER OF READS OF FILE LUINTA           1
  FINAL ORDERING...
  NUMBER OF ATOMIC INTEGRALS READ          0
  NUMBER OF ORDERED INTEGRALS        1622726
  NUMBER OF BIN WRITES                     0

     NUMBER OF INTEGRALS PER SYMMETRY BLOCK(   1 BLOCKS)
        1622726
  CPU/IO TIME (SEC), TOTAL:    0.0         0.0

 
 **************************************************
        SUPERMATRIX CONVERSION PROGRAM
        IBM VERSION 3, P-AA M 88-04-21
 **************************************************
        JOB RUN ON and AT Sun Oct 17 15:54:41 2021



  rec.unit = 4 bytes
  Data from MolDat? "y" or "n": 
  Word for MolDat: 
  PRINT LEVEL            1

 NO SYMMETRY USED. NR OF BASIS FUNCTIONS IS  80
  LC.LE.0 ! (A.T.)
 
  SYMMETRY OF EACH BFN, AND OFFSET ITAB
  TO FORM PAIR INDICES:
 
      I:     1     2     3     4     5     6     7     8     9    10
   ISYM:     1     1     1     1     1     1     1     1     1     1
   ITAB:     0     1     3     6    10    15    21    28    36    45
 
      I:    11    12    13    14    15    16    17    18    19    20
   ISYM:     1     1     1     1     1     1     1     1     1     1
   ITAB:    55    66    78    91   105   120   136   153   171   190
 
      I:    21    22    23    24    25    26    27    28    29    30
   ISYM:     1     1     1     1     1     1     1     1     1     1
   ITAB:   210   231   253   276   300   325   351   378   406   435
 
      I:    31    32    33    34    35    36    37    38    39    40
   ISYM:     1     1     1     1     1     1     1     1     1     1
   ITAB:   465   496   528   561   595   630   666   703   741   780
 
      I:    41    42    43    44    45    46    47    48    49    50
   ISYM:     1     1     1     1     1     1     1     1     1     1
   ITAB:   820   861   903   946   990  1035  1081  1128  1176  1225
 
      I:    51    52    53    54    55    56    57    58    59    60
   ISYM:     1     1     1     1     1     1     1     1     1     1
   ITAB:  1275  1326  1378  1431  1485  1540  1596  1653  1711  1770
 
      I:    61    62    63    64    65    66    67    68    69    70
   ISYM:     1     1     1     1     1     1     1     1     1     1
   ITAB:  1830  1891  1953  2016  2080  2145  2211  2278  2346  2415
 
      I:    71    72    73    74    75    76    77    78    79    80
   ISYM:     1     1     1     1     1     1     1     1     1     1
   ITAB:  2485  2556  2628  2701  2775  2850  2926  3003  3081  3160
 
  ORDERED INTEGRAL FILE.
  INDEXED SUPERMATRIX FILE.
  NR OF INDICES/INTEGER*4 :           1
  HALF-SIZED DIAGONAL (IJIJ) ELEMENTS.
  MAX POSSIBLE NR OF INTEGRALS:   5250420
       OF SUPERMATRIX ELEMENTS:   5250420
  THRESHOLD FOR INTEGRALS: 0.0D+00
     D:O, S.M. ELEMENTS  : 0.1D-09
 
         NR OF INTEGRALS READ:  10497600
         NR OF INTEGRALS USED:   5250420
  NR OF CONTRIBUTIONS TO S.M.:   2438538
  CPU/IO TIME (SEC):    1.0         0.0
 
 SUPERMATRIX ELEMENTS WRITTEN:    914308
   TOTAL DAFILE UNITS WRITTEN:  46850048
  CPU/IO TIME (SEC), TOTAL:    1.0         0.0
 
 **************************************************
 

        JOB RUN ON and AT Sun Oct 17 15:54:42 2021



  rec.unit = 4 bytes
  Data from MolDat? "y" or "n": 
  Word for MolDat: 
 OPTION:
'                                                           
 *****************************************************************
 CLOSED SHELL SCF PROGRAM:   IBM VERSION 87-10-23 FROM MOLCAS

 *****************************************************************


 
 
 OCCUPATION NUMBERS
 SYMMETRY  1
  2.000 2.000 2.000 2.000 2.000 2.000 2.000 2.000 2.000
 
     This file was automatically generated by the FSCC driver            
     Sun Oct 17 15:54:40 2021                                            
 
 
 Input given to program:
 
 Print level=  1
 No orbitals printed
 All orbs punched on SCFORB (for Monstr & RAS)
 Starting vectors from scratch
 
 Maximum number of iterations       50
 Energy threshold                  0.10000000E-11
 Density matrix threshold          0.10000000E-09
 
 DIIS extrapolation of SCF procedure
 S-component of d-orbitals deleted
 P-component of f-orbitals deleted
 S and d-components of g-orbitals deleted
 Threshold for linear dependence   0.99999997E-09
 
 
 Number of symmetries                1
 Total number of orbitals           59
 Number of first orb-l in Isym:      1
 Total number of bfns               80
 Number of occupied orbitals         9
 Number of frozen orbitals           0
 
 
 
 
ITER       ENERGY          ONE-EL          TWO-EL           DMAT            CAUF
   1    -79.15796972   -172.71464526     88.55667554      0.68836257      1.00000000
   2    -81.70482650   -151.86864641     65.16381991      0.42549742      0.26987006
   3    -85.98822504   -164.85233267     73.86410763      0.30864156      1.00000000
 CAUF (0.02) TOO SMALL, -> 0.15
   4    -85.58389771   -154.57417869     63.99028097      0.05799913      0.01658378
   5    -85.85629411   -158.38207728     67.52578317      0.04438354      0.36479255
   6    -85.89014854   -159.01321172     68.12306318      0.04912946      0.17678611
   7    -85.90516355   -159.29411921     68.38895566      0.06299586      0.22089049
   8    -86.02937229   -162.45476212     71.42538983      0.78825906     -0.54043095
   9    -85.99437464   -160.71706979     69.72269515      0.33581733      0.13570400
  10    -86.00960890   -164.28400596     73.27439706      0.59867280     -1.00406147
  11    -85.99635275   -166.62008901     75.62373626      0.34694542      0.01854159
  12    -86.02781685   -164.14196711     73.11415026      0.57181292      0.03075072
  13    -86.02462053   -164.10294031     73.07831978      1.02857911      0.02107302
  14    -86.02851287   -165.52935600     74.50084313      0.67740854      0.02122639
  15    -86.01629471   -164.49034084     73.47404613      0.54529141      0.01366851
  16    -86.01946666   -165.04031050     74.02084384      0.29725417      0.02018139
  17    -86.00655052   -164.94366135     73.93711083      0.44163042      0.01847986
  18    -85.99733840   -164.47871259     73.48137419      0.46473762      0.01122241
  19    -86.00014962   -164.05999048     73.05984086      0.66242988      0.01398431
  20    -85.99666421   -164.99911269     74.00244848      0.97957778      0.01276796
  21    -86.02676348   -164.45149252     73.42472904      0.92522338      0.01299538
  22    -85.99066065   -163.66191151     72.67125086      0.64590402      0.01239077
  23    -85.99822665   -165.19623594     74.19800929      0.29284912      0.03075095
  24    -86.00809255   -164.74603488     73.73794232      1.08076721      0.02111289
  25    -85.99938767   -166.04781314     75.04842546      0.34196604      0.01470895
  26    -86.01857175   -163.41174742     72.39317566      0.38044323      0.03648152
  27    -86.00244372   -165.44359207     74.44114834      0.24832594      0.02899705
  28    -86.01937082   -164.34261426     73.32324344      0.61312943      0.02916588
  29    -86.01008844   -165.72992215     74.71983372      0.41327991      0.02780775
  30    -86.03103027   -164.49135842     73.46032814      1.02730150      0.02636138
  31    -86.02163562   -166.40217939     75.38054377      0.48556566      0.02990282
  32    -86.02923150   -164.18538468     73.15615318      0.72908085      0.02931354
  33    -86.01384963   -165.90648648     74.89263686      0.38714126      0.02609448
  34    -86.03655218   -164.20586776     73.16931558      0.64119823      0.03134894
  35    -86.01049405   -165.84658868     74.83609463      0.74566479      0.03977023
  36    -86.02695840   -164.82886899     73.80191059      1.09734143      0.02837333
  37    -86.01948367   -165.91376501     74.89428133      0.73327600      0.02880732
  38    -86.03865104   -164.31390231     73.27525128      0.91460748      0.02845645
  39    -86.03209732   -165.91483913     74.88274181      0.91364411      0.03903988
  40    -86.05233741   -164.61329157     73.56095416      0.97349879      0.02509303
  41    -86.02190299   -165.96563629     74.94373331      0.69892015      0.03017058
  42    -86.07293834   -164.25549993     73.18256159      0.90534537      0.03741856
  43    -86.01335968   -166.03546555     75.02210587      0.76875304      0.03524717
  44    -86.02657952   -164.47080040     73.44422088      0.74486781      0.03409039
  45    -86.01101234   -166.13727541     75.12626306      0.89966118      0.04192609
  46    -86.06141958   -164.91489582     73.85347623      0.89622642      0.03717872
  47    -86.01766267   -166.18391078     75.16624811      0.98360392      0.04212078
  48    -86.07023669   -165.30547798     74.23524128      0.89922644      0.03769987
  49    -86.07201821   -165.73836201     74.66634380      0.78367063      0.04051744
  50    -86.03359710   -165.13887032     74.10527322      0.85561530      0.03909205
 
 
 
 ******   FINAL PRINTOUT   ******
 ******        FINAL       ******
 
 TOTAL ENERGY              -86.07964404
 - " - in ev:            -2342.36914573
 1/cm (*10**5)            -188.91899477
 ONE ELECTRON ENERGY      -165.09918802
 TWO ELECTRON ENERGY        74.01954398
 NUCLEAR REPULSION           5.00000000
 KINETIC ENERGY             54.62407191
 VIRIAL THEOREM              1.57585550
 DMAT DIFFERENCE             2.06777299
 MAX D IN MO                 0.85561530


          ORBITAL ENERGIES
          ----------------

 SYMMETRY  1

     -4.47077 (2.00)     -0.81866 (2.01)     -0.54005 (-.11)     -0.53883 (-.08)     -0.53775 (0.01)
     -0.53699 (-.05)     -0.53540 (2.23)     -0.51691 (0.82)     -0.50056 (0.39)
     -0.46621 (0.00)     -0.46452 (0.00)     -0.44856 (0.00)     -0.42649 (0.00)     -0.42614 (0.00)
     -0.41733 (0.00)     -0.41423 (0.00)      1.03675 (0.00)      1.05511 (0.00)      1.05917 (0.00)
      1.05926 (0.00)      1.06018 (0.00)      1.06932 (0.00)      1.06946 (0.00)      1.07188 (0.00)
      1.07257 (0.00)      1.91007 (0.00)      1.94831 (0.00)      1.94849 (0.00)      2.47604 (0.00)
      2.47641 (0.00)      2.47846 (0.00)      2.49084 (0.00)      2.49084 (0.00)      4.20592 (0.00)
      4.20593 (0.00)      4.21864 (0.00)      4.21869 (0.00)      4.23129 (0.00)      4.23192 (0.00)
      4.25476 (0.00)      4.26536 (0.00)      4.28840 (0.00)      6.58856 (0.00)      6.58860 (0.00)
      6.60453 (0.00)      6.60547 (0.00)      6.61535 (0.00)      6.62548 (0.00)      6.64192 (0.00)
     41.27206 (0.00)    174.83314 (0.00)    174.83323 (0.00)    174.83511 (0.00)    174.83609 (0.00)
    174.83654 (0.00)    255.29791 (0.00)    255.29903 (0.00)    255.30124 (0.00)    673.35465 (0.00)


                    GROSS ATOMIC POPULATIONS PER CENTER AND TYPE
                    --------------------------------------------
 
          Hg      H   
  S     2.0014  0.0003
  X     0.0555 -0.0003
  Y     0.0846 -0.0003
  Z     0.0379  0.0043
  XX    0.3163 -0.0020
  XY    0.0656  0.0000
  XZ   -0.1607  0.0019
  YY    0.3185 -0.0020
  YZ   -0.1720  0.0019
  ZZ    1.1299  0.0134
  XXX   0.7956  0.0000
  XXY   1.5477  0.0000
  XXZ   1.6381  0.0000
  XYY   1.5494  0.0000
  XYZ   1.9593  0.0000
  XZZ   1.7546  0.0000
  YYY   0.7999  0.0000
  YYZ   1.6114  0.0000
  YZZ   1.7409  0.0000
  ZZZ   0.9093  0.0000
  XXXX  0.0004  0.0000
  XXXY -0.0001  0.0000
  XXXZ -0.0007  0.0000
  XXYY  0.0002  0.0000
  XXYZ -0.0003  0.0000
  XXZZ -0.0015  0.0000
  XYYY -0.0001  0.0000
  XYYZ -0.0003  0.0000
  XYZZ -0.0002  0.0000
  XZZZ  0.0008  0.0000
  YYYY  0.0005  0.0000
  YYYZ -0.0006  0.0000
  YYZZ -0.0014  0.0000
  YZZZ  0.0008  0.0000
  ZZZZ  0.0023  0.0000
 Total 17.9828  0.0172
 
          Total electronic charge=   18.000000
 
 Total Cpu time:   0.00 Seconds
 Total I/O time:   0.00 Seconds
 
 ***************************************
 ******  FILE STATUS FROM FASTIO  ******
 ***************************************
 
         LU  NAME      XTENT     LRECL
          2 ONEINT        23       4KB
         18 SUPMAT      3640       4KB
 
 Routine     Calls     Cpu     I/O
  SCF           1      0.0     0.0
  READIN        1      0.0     0.0
  SORB          1      0.0     0.0
  DMAT         51      0.0     0.0
  PMAT         51      0.0     0.0
  INTERP        4      0.0     0.0
  NEWORB       51      0.0     0.0
  MODENS       50      0.0     0.0
  PRINT        51      0.0     0.0
  DIIS         47      0.0     0.0
 


		*************************************************************
		**                     END OF FSCC OUTPUT                  **
		*************************************************************

FSCC terminated normally at Sun Oct 17 15:54:42 2021
Total run time: 0 days 0 hours 0 minutes 2 seconds 36 milliseconds
